### DISCUSSION TITLE
Welcome to PokemonGo-Automate-Flows Discussions!

### DISCUSSION THREAD

**Discussion by chrisisth** [chrisisth/PokemonGo-Automate-Flows#1] [Announcements]
## 👋 Welcome to the Automate Flows for Pokemon Go Players Discussion Board!

This discussion board is a place for users of the Automate flows for Pokemon Go players to connect and collaborate. I encourage you to:

- Ask questions about the Automate flows.
- Share your experiences and ideas.
- Engage with other community members.
- Welcome others and be open-minded. Remember, this is a community we build together 💪.

### Rules for the Discussion Board

1. **Be respectful**: Every person on this board has the right to feel comfortable. Avoid using offensive language and respect the opinions and ideas of others.
2. **Stay on topic**: Make sure your posts are relevant to Automate flows for Pokemon Go. Off-topic posts may be removed.
3. **No spam**: Refrain from posting the same message in multiple threads. Also, avoid advertising or self-promotion.
4. **Protect privacy**: Do not share personal information, yours or others, such as addresses, phone numbers, emails, etc. Respect the privacy of others and keep your account secure.

Have fun! 🎉

**Comment by Neoplasma** [comment]
Hi ive downloaded the auto trader file but it wont trade.
It opens the trade window then it click the pokemon but then it doesnt press next.

I dont know how i can reach you.

**Reply by chrisisth** [reply]
Ah damn seems like the timings are off again, i'll look into it.

**Reply by chrisisth** [reply]
Hey, I have revised the trade flow, please try it out and let me know if it works.

**Comment by madmanDAS** [comment]
Hi there, having issues with the gift managment, it seems to click on the friend profile instead of a gift, how would I go about calibrating the touch for my device? (Noob at automate with a full pogo list looking to make life easier)

**Reply by chrisisth** [reply]
Hey, sorry it took me so long to answer.

I made some updates.
I removed the click on the Niantic balloon icon because it didn’t work as intended. My goal was to prevent misclicks when no present is available to open.
I also improved the swiping after each gift is opened.
However, it may still not work correctly on your device. To debug this, follow this:


By enabling Pointer location in the development options you can see where the script will press when its running.
![351364044-dc902e9e-8073-45c5-9be9-c33d3384cdbc](https://github.com/user-attachments/assets/c53027b1-b51f-48ae-98d6-89a910f86e1e)
